### PR TITLE
update doc for waveflow

### PR DESCRIPTION
**Pull request by iclementine**
1. update doc for waveflow
2. fix: import normalizer into frontend.

### CODE REVIEW

File: parakeet/models/waveflow.py [hunk 1]
```diff
@@ -9,27 +9,56 @@
...
+        The input tensor.
+        
+    n_group : int
+        The size of a group.
 
-    Returns:
-        Tensor: shape(*, time_steps // n_group, group), folded tensor.
+    Returns
+    ---------
+    Tensor : [shape=(`*, time_steps // n_group, group)]
```
Comment (lfchener) [inline parakeet/models/waveflow.py]: \\* not `*?

File: parakeet/models/waveflow.py [hunk 2]
```diff
@@ -155,13 +217,21 @@ def start_sequence(self):
...
 
-        Returns:
-            res (Tensor): shape(batch_size, channel, 1, width), the residual output.
-            res (Tensor): shape(batch_size, channel, 1, width), the skip output.
+        Returns
+        -------
+        res : Tensor [shape=(batch_size, channel, 1, width)]
+            A row of the the residual output.
+            
+        res : Tensor [shape=(batch_size, channel, 1, width)]
```
Comment (lfchener) [inline parakeet/models/waveflow.py]: output name is skip not res?

File: parakeet/models/waveflow.py [hunk 3]
```diff
@@ -228,20 +328,29 @@ def forward(self, x, condition):
...
 
-        Returns:
-            Tensor: shape(batch_size, channel, 1, width), the output, which is 
-                an aggregation of all the skip outputs.
+        Returns
+        -------
+        res : Tensor [shape=(batch_size, channel, 1, width)]
+            A row of the the residual output.
+            
+        res : Tensor [shape=(batch_size, channel, 1, width)]
```
Comment (lfchener) [inline parakeet/models/waveflow.py]: skip?

File: parakeet/models/waveflow.py [hunk 4]
```diff
@@ -252,12 +361,29 @@ def add_input(self, x_row, condition_row):
...
 class Flow(nn.Layer):
-    """
-    A bijection (Reversable layer) that transform a density of latent variables 
-    p(Z) into a complex data distribution p(X).
+    """A bijection (Reversable layer) that transform a density of latent 
+    variables p(Z) into a complex data distribution p(X).
 
-    It's a auto regressive flow. The `forward` method implements the probability
-    density estimation. The `inverse` method implements the sampling.
+    It's an auto regressive flow. The `forward` method implements the
```
Comment (lfchener) [inline parakeet/models/waveflow.py]: `  -> ``

File: parakeet/models/wavenet.py [hunk 5]
```diff
@@ -326,22 +349,32 @@ def forward(self, x, condition=None):
         return skip_connections
 
     def start_sequence(self):
-        """Prepare the ResidualNet to generate a new sequence. This method should be called before starting calling `add_input` multiple times.
+        """Prepare the ResidualNet to generate a new sequence. This method 
+        should be called before starting calling `add_input` multiple times.
```
Comment (lfchener) [inline parakeet/models/wavenet.py]: ``

File: parakeet/models/wavenet.py [hunk 6]
```diff
@@ -326,22 +349,32 @@ def forward(self, x, condition=None):
...
+            
+        condition : Tensor, optional [shape=(B, C_cond)]
+            Condition for a step. Defaults to None.
 
-        Returns:
-            skip_connection (Tensor): shape(B, C_res), dtype float32, the output for a step.
+        Returns
+        ----------            
+        Tensor [shape=(B, C)]
+            T he skip connection for a step. This output is accumulated with
```
Comment (lfchener) [inline parakeet/models/wavenet.py]: The?

File: parakeet/models/wavenet.py [hunk 7]
```diff
@@ -426,19 +493,28 @@ def forward(self, x, condition=None):
...
+            
+        condition : Tensor, optional [shape=(B, C_cond)]
+            A step of the upsampled condition. Defaults to None.
 
-        Returns:
-            Tensor: shape(B, C_output), dtype float32, the parameter of the output distributions.
+        Returns
+        --------
+        Tensor: [shape=(B, C_output)]
+            A steo of the parameters of the output distributions.
```
Comment (lfchener) [inline parakeet/models/wavenet.py]: step?

File: parakeet/models/wavenet.py [hunk 8]
```diff
@@ -568,26 +692,37 @@ def sample_from_mog(self, y):
...
-        """compute the loss where output distribution is a mixture of Gaussians.
+        """Compute the loss given the output distribution and the target.
 
-        Args:
-            y (Tensor): shape(B, T, C_output), dtype float32, the parameterd of the output distribution.
-            t (Tensor): shape(B, T), dtype float32, the target audio. Note that the target's corresponding time index is one step ahead of the output distribution. And output distribution whose input contains padding is neglected in loss computation.
+        Parameters
+        ----------
+        y : Tensor [shape=(B, T, C_output)]
+            The parameterd of the output distribution.
```
Comment (lfchener) [inline parakeet/models/wavenet.py]: parameters?

File: parakeet/models/waveflow.py (hunk 1 above)
Comment (iclementine) [reply parakeet/models/waveflow.py]: done

File: parakeet/models/waveflow.py (hunk 2 above)
Comment (iclementine) [reply parakeet/models/waveflow.py]: done

File: parakeet/models/waveflow.py (hunk 3 above)
Comment (iclementine) [reply parakeet/models/waveflow.py]: done

File: parakeet/models/waveflow.py (hunk 4 above)
Comment (iclementine) [reply parakeet/models/waveflow.py]: done

File: parakeet/models/wavenet.py (hunk 5 above)
Comment (iclementine) [reply parakeet/models/wavenet.py]: done

File: parakeet/models/wavenet.py (hunk 6 above)
Comment (iclementine) [reply parakeet/models/wavenet.py]: done

File: parakeet/models/wavenet.py (hunk 7 above)
Comment (iclementine) [reply parakeet/models/wavenet.py]: done

File: parakeet/models/wavenet.py (hunk 8 above)
Comment (iclementine) [reply parakeet/models/wavenet.py]: done

**Review by lfchener — APPROVED** [review]
LGTM

**Pull request merged**